### ISSUE TITLE
Problem with TMVAClassifier

### ISSUE THREAD

**Issue by Leensman** [yandex/rep#38]
After REP installation from here, I've met the following problem with TMVAClassifier fitting: I'm trying to train TMVAClassifier, and IOError raises after following strings: 
"<i> baseline = TMVAClassifier(method='kBDT', features=variables, BoostType='Grad', NTrees=40, Shrinkage=0.01, MaxDepth=7, UseNvars=6, nCuts=-1) features=variables)

baseline.fit(train, train['signal'])</i>"

<b>Stacktrace</b> is next:
<i>
IOError Traceback (most recent call last) in () 3 UseNvars=6, nCuts=-1) 4 # baseline = TMVAClassifier(method='kBDT', NTrees=50, Shrinkage=0.05, features=variables) ----> 5 baseline.fit(train, train['signal'])

/usr/local/lib/python2.7/dist-packages/rep-0.6.3-py2.7.egg/rep/estimators/tmva.pyc in fit(self, X, y, sample_weight) 288 self.factory_options = '{}:AnalysisType=Multiclass'.format(self.factory_options) 289 --> 290 return self._fit(X, y, sample_weight=sample_weight) 291 292 def predict_proba(self, X):

/usr/local/lib/python2.7/dist-packages/rep-0.6.3-py2.7.egg/rep/estimators/tmva.pyc in _fit(self, X, y, sample_weight, model_type) 104 add_info = _AdditionalInformation(directory, model_type=model_type) 105 try: --> 106 self._run_tmva_training(add_info, X, y, sample_weight) 107 finally: 108 self._remove_tmp_directory(directory)

/usr/local/lib/python2.7/dist-packages/rep-0.6.3-py2.7.egg/rep/estimators/tmva.pyc in _run_tmva_training(self, info, X, y, sample_weight) 134 xml_filename = os.path.join(info.directory, 'weights', 135 '{job}_{name}.weights.xml'.format(job=info.tmva_job, name=self._method_name)) --> 136 with open(xml_filename, 'r') as xml_file: 137 self.formula_xml = xml_file.read() 138

IOError: [Errno 2] No such file or directory: '/home/artem/Documents/IPython Notebooks/CERN + Yandex/Original Baseline/flavours-of-physics-start/tmp0Fhtqe/weights/TMVAEstimation_REP_Estimator.weights.xml'
</i>

As I found, weights/ folder was created outside of temporary folder instead created inside in last one. It causes the error above.

ROOT 5.34, Python 2.7, GCC 4.8, Ubuntu 14.04 LTS (x64). All requirenments for REP were installed successfully (from requirenments.txt)

**Comment by tata-antares** [comment]
Can you run in the same notebook the following code:

```python
import tempfile
import os
name = tempfile.mkdtemp(dir=os.getcwd())
```
and then in other cell:
```
!cd $name
```

I want to check that process has access to the temporary folder.

**Comment by Leensman** [comment]
The result is:
/bin/sh: 1: cd: can't cd to /home/artem/Documents/IPython

**Comment by tata-antares** [comment]
Run:

```
ls -la /home/artem/Documents/
```
And say, how you run IPython (what user)?

**Comment by Leensman** [comment]
ls -la /home/artem/Documents/ works OK. IPython was launched from admin user, but admin not in sudo group.

**Comment by tata-antares** [comment]
Write output of ls -la /home/artem/Documents/ to see the rights of  /home/artem/Documents/IPython

**Comment by Leensman** [comment]
итого 13172
drwxr-xr-x  5 artem artem    4096 сент. 27 16:27 ./
drwxr-xr-x 27 artem artem    4096 сент. 27 15:26 ../
-rw-r-----  1 artem artem 2326130 сент. 15 18:37 EkmanPol_PsihologiyaLzhi.pdf
-rw-r-----  1 artem artem 3829480 сент. 17 13:41 evaluating-machine-learning-models.pdf
drwx------  4 artem artem    4096 сент.  8 21:49 IPython Notebooks/
-rw-r-----  1 artem artem 3144056 сент. 11 19:43 K_V_Vorontsov_Mat_metody_obuchenia_po_pretsede.pdf
drwxrwxr-x  3 artem artem    4096 сент. 27 14:31 SHAD/
-rw-r-----  1 artem artem 4147233 сент. 15 18:37 Telling Lies - Clues to Deceit in the Marketplace, Politics, and Marriage.pdf
drwxrwxr-x  3 artem artem    4096 сент. 27 16:27 tmp/

**Comment by tata-antares** [comment]
The problem is your path: IPython Notebooks/. It contains space, that is why command 'cd {folder}' doesn't work, and weights is created outside the temp folder. 

Thanks, will fix this error in next release.

At this moment, rename this folder without spaces, or move starter kit notebooks into other place.

**Comment by Leensman** [comment]
Ok,  thanks!

**Comment by tata-antares** [comment]
fix in commit 1ac0a59cc3c568646fbf360c3dc0b62a63471090
